### DISCUSSION TITLE
Search two keys at a time

### DISCUSSION THREAD

**Discussion by gkcsabado** [TarekRaafat/autoComplete.js#340] [Q&A] [answered]
Hello, is it possible two search for two keys at a time in a single field? I have a data that consists of zipcodes, cities and abbr state
Sample data:
```
[
  {"zip":"11234", "city":"Brooklyn", "state":"NY"},
  {"zip":"92804", "city":"Anaheim", "state":"CA"},
  {"zip":"11226", "city":"Brooklyn", "state":"NY"},
  {"zip":"93307", "city":"Bakersfield", "state":"CA"},
  {"zip":"10453", "city":"Bronx", "state":"NY"},
  ...
]
```

I would like to know if I can search for something in the field like **Brooklyn, NY** or **Brooklyn NY** and will display the results that contains the city Brooklyn with the state NY. In this case, it should not display Bronx because even if the state is NY, the city would not matched up.

**Edit:** My current progress so far is when I typed in **"Brooklyn,"**, after the comma, it will return no results as it only searching for one key which is city. Also when I typed in **"Brooklyn N"**, it will also return nothing.

**Comment by gkcsabado** [comment, marked as answer]
Already resolved it by removing data.keys and instead use the searchEngine option (https://tarekraafat.github.io/autoComplete.js/#/configuration?id=searchengine-optional) so that I could return concatenated data (city and state) that depends on custom conditions.

**Reply by baltata** [reply]
Hello, would you be kind enough to share yur code ? We are having the same issue here...
